packages/wallet/src/views/Wallet.jsx

```diff
@@ -7,575 +7,153 @@
-                      Tokenomics
-                    </p>
-                    <div className='mt-6'>
-                      <button
-                        className='bg-pink-500 text-white w-8 h-8 rounded-full outline-none focus:outline-none mr-1 mb-1'
-                        type='button'
-                      >
-                        <i className='fab fa-dribbble' />
-                      </button>
-                      <button
-                        className='bg-red-600 text-white w-8 h-8 rounded-full outline-none focus:outline-none mr-1 mb-1'
-                        type='button'
-                      >
-                        <i className='fab fa-google' />
-                      </button>
-                      <button
-                        className='bg-blue-400 text-white w-8 h-8 rounded-full outline-none focus:outline-none mr-1 mb-1'
-                        type='button'
-                      >
-                        <i className='fab fa-twitter' />
-                      </button>
-                      <button
-                        className='bg-gray-800 text-white w-8 h-8 rounded-full outline-none focus:outline-none mr-1 mb-1'
-                        type='button'
-                      >
-                        <i className='fab fa-instagram' />
-                      </button>
-                    </div>
                   </div>
                 </div>
               </div>
             </div>
-          </div>
-        </section>
-
-        <section className='pb-20 relative block bg-gray-900'>
-          <div
-            className='bottom-auto top-0 left-0 right-0 w-full absolute pointer-events-none overflow-hidden -mt-20'
-            style={{ height: '80px' }}
-          >
-            <svg
-              className='absolute bottom-0 overflow-hidden'
-              xmlns='http://www.w3.org/2000/svg'
-              preserveAspectRatio='none'
-              version='1.1'
-              viewBox='0 0 2560 100'
-              x='0'
-              y='0'
-            >
-              <polygon
-                className='text-gray-900 fill-current'
-                points='2560 0 2560 100 0 100'
-              />
-            </svg>
-          </div>
-
-          <div className='container mx-auto px-4 lg:pt-24 lg:pb-64'>
-            <div className='flex flex-wrap text-center justify-center'>
-              <div className='w-full lg:w-6/12 px-4'>
-                <h2 className='text-4xl font-semibold text-white'>
-                  Buidl something
-                </h2>
-                <p className='text-lg leading-relaxed mt-4 mb-4 text-gray-500'>
-                  We enjoy buidling together.
+            <div className='pt-20'>
+              <h1 className='font-heading text-5xl text-primary pb-6'>Supporting Supporters</h1>
+              <div className='font-secondary'>
+                <p className='pb-10'>
+                  At Kernel, we really care about{' '}
```

Comment:
> {' '}

This is a bit unusual and you should remove it. If you ever need a non breaking space, you can use HTML code `&nbsp;`.

In this particular case, what you really want is some margin and you can add it to the `a` element like so -

```
<a className='px-2' href='https://www.kernel.community/en/conversation/reciprocity' target='_blank' rel='noreferrer'>
```

See Tailwind reference here -
https://tailwindcss.com/docs/padding#add-horizontal-padding